docs/DeveloperGuide.md

```diff
@@ -385,6 +388,145 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
     * 3q1. EstateEase shows an error message regarding the entry of incorrect priority level. <br>
       Use case resumes from step 2.
 
+**Use case: UC00 - Add home-buyer to user’s contact list**
```

Comment:
This one too.

Reply:
Yeap updated the numbering of the use case. Thanks for reminding.